.github/workflows/trigger-tests-on-nightly-packages.yml

```diff
@@ -0,0 +1,12 @@
+name: Trigger tests for nightly APT packages
+
+on:
+  schedule:
+    - cron: '00 3 * * *'
+
+jobs:
+  test_nightly_packages:
+    uses: ./.github/workflows/test-packages.yml
```

Comment:
Note: we should review all of our "on schedule" workflows and consider adding `if: github.repository_owner == 'syslog-ng'` in order not to run them in forks.

Reply:
Thanks for the review comment, agreed with this.
Fixed

Reply:
Sorry, I was not precise enough, something like this will do the trick:
`if: (github.event_name == 'schedule' && github.repository_owner == 'syslog-ng') || (github.event_name != 'schedule')`

Basically, we
- should allow starting all of our scheduled jobs manually, even in forks
- should not run those jobs on schedule in forks (we have 400 forks, fortunately only ~5% of them is updated to run our workflows)